serverlessworkflow/modules/ROOT/pages/core/cncf-serverless-workflow-specification-support.adoc

```diff
@@ -38,129 +38,125 @@ specification.
 |===
 | Feature | Status | Reference
 
-| <<states>>
+| <<compensation>>
 | emoji:full_moon[]
-| link:{spec_doc_url}#workflow-states[Workflow States]
+| link:{spec_doc_url}#Workflow-Compensation[Workflow Compensation]
 
-| <<functions>>
-| emoji:last_quarter_moon[]
-| link:{spec_doc_url}#Function-Definition[Function Definition]
+| <<constants>>
+| emoji:full_moon[]
+| link:{spec_doc_url}#workflow-constants[Workflow Constants]
+
+| <<error_handling>>
+| emoji:full_moon[]
+| link:{spec_doc_url}#Workflow-Error-Handling[Workflow Error Handling]
 
 | <<events>>
 | emoji:last_quarter_moon[]
 | link:{spec_doc_url}#Event-Definition[Event Definition]
 
-| <<workflow_data>>
-| emoji:full_moon[]
-| link:{spec_doc_url}#Workflow-Data[Workflow Data]
-
 | <<expressions>>
 | emoji:full_moon[]
 | link:{spec_doc_url}#Workflow-Expressions[Workflow Expressions]
 
-| <<error_handling>>
-| emoji:full_moon[]
-| link:{spec_doc_url}#Workflow-Error-Handling[Workflow Error Handling]
+| <<functions>>
+| emoji:last_quarter_moon[]
+| link:{spec_doc_url}#Function-Definition[Function Definition]
 
 | <<retries>>
 | emoji:construction[]
 | link:{spec_doc_url}#Retry-Definition[Retry Definition]
 
-| <<timeouts>>
-| emoji:last_quarter_moon[]
-| link:{spec_doc_url}#workflow-timeouts[Workflow Timeouts]
-
-| <<compensation>>
+| <<secrets>>
 | emoji:full_moon[]
-| link:{spec_doc_url}#Workflow-Compensation[Workflow Compensation]
+| link:{spec_doc_url}#workflow-secrets[Workflow Secrets]
 
-| <<constants>>
+| <<workflow_data>>
 | emoji:full_moon[]
-| link:{spec_doc_url}#workflow-constants[Workflow Constants]
+| link:{spec_doc_url}#Workflow-Data[Workflow Data]
 
-| <<secrets>>
+| <<states>>
 | emoji:full_moon[]
-| link:{spec_doc_url}#workflow-secrets[Workflow Secrets]
+| link:{spec_doc_url}#workflow-states[Workflow States]
+
+| <<timeouts>>
+| emoji:last_quarter_moon[]
+| link:{spec_doc_url}#workflow-timeouts[Workflow Timeouts]
 |===
 
 [[states]]
 == States
 
 The link:{spec_doc_url}#parallel-state[Parallel State] of the workflow states feature works in a single thread. This means that a Parallel State does not create one thread per branch, simulating an actual parallel behavior.
 
- If an exclusive property is set to `false`, you should not use the link:{spec_doc_url}#event-state[Event State] of the workflow states feature as the starting state. In case, if it is specified that way, then it will behave as if an exclusive property was set to `true`. 
+If an `exclusive` property is set to `false`, you should not use the link:{spec_doc_url}#Event-State[Event State] of the workflow states feature as the starting state. In case that  it is specified that way, it will behave as if an `exclusive` property was set to `true`. 
```

Comment:
```suggestion
If an `exclusive` property is set to `false`, you should not use the link:{spec_doc_url}#Event-State[Event State] of the workflow states feature as the starting state. In case that it is specified that way, it will behave as if an `exclusive` property was set to `true`. 
```